cmd/incusd/instance_access.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+
+	"github.com/gorilla/mux"
+
+	internalInstance "github.com/lxc/incus/v6/internal/instance"
+	"github.com/lxc/incus/v6/internal/server/request"
+	"github.com/lxc/incus/v6/internal/server/response"
+)
+
+// swagger:operation GET /1.0/instances/{name}/access
+//
+//	Get who has access to an instnace
+//
+//	---
+//	produces:
+//	  - application/json
+//	parameters:
+//	  - in: query
+//	    name: project
+//	    description: Project name
+//	    type: string
+//	responses:
+//	  "200":
+//	    description: Access
+//	    schema:
+//	      type: list
+//	      description: Sync response
+//	      properties:
+//	        type:
+//	          type: object
+//	          description: info for each access entry
+//	          properties:
+//		        identifier:
+//		          type: string
+//		          description: certificate fingerprint
+//		          example: 636b69519d27ae3b0e398cb7928043846ce1e3842f0ca7a589993dd913ab8cc9
+//		        role:
+//		          type: string
+//		          description: the role associated with the certificate
+//		          example: admin
+//		        provider:
+//		          type: string
+//		          description: Which authorization method the certificate uses
+//		          example: tls
```

Comment:
It's usually not required to fill that stuff in here as we can just reference the api struct (`Access`) and have swagger pull the model from that instead.

So you'd have something like this instead:
```
//  "200":
//    description: Access
//    schema:
//      type: object
//      description: Sync response
//      properties:
//        type:
//          type: string
//          description: Response type
//          example: sync
//        status:
//          type: string
//          description: Status description
//          example: Success
//        status_code:
//          type: integer
//          description: Status code
//          example: 200
//        metadata:
//          $ref: "#/definitions/Access"
```